docs/Oracle Jobs/Job Types/direct_request.md

```diff
@@ -96,3 +99,84 @@ encode_tx       [type=ethabiencode
 
 submit_tx  [type=ethtx to="0x613a38AC1659769640aaE063C651F48E0250454C" data="$(encode_tx)"]
 ```
+
+**Multi word example**
+
```

Comment:
Can we get a short description sentence to explain the difference for multi vs single?